pkg/es/config/config.go

```diff
@@ -465,20 +546,20 @@ func GetHTTPRoundTripper(c *Configuration, logger *zap.Logger) (http.RoundTrippe
 	}
 
 	token := ""
-	if c.TokenFilePath != "" {
-		if c.AllowTokenFromContext {
+	if c.Authentication.BearerTokenAuthentication.FilePath != "" {
+		if c.Authentication.BearerTokenAuthentication.AllowFromContext {
 			logger.Warn("Token file and token propagation are both enabled, token from file won't be used")
 		}
-		tokenFromFile, err := loadTokenFromFile(c.TokenFilePath)
+		tokenFromFile, err := loadTokenFromFile(c.Authentication.BearerTokenAuthentication.FilePath)
```

Comment:
weird that despite the PR title #4342 there is no "reloading" of the file, only at init time (whereas password does get reloaded). But since nobody complained ...

I don't see much documentation on the website about how this token is supposed to work, specifically any pointers to the documentation for the actual storage (e.g. ES or hosted ES). Might have been some references in the original issues but hard to find them.

Reply:
I guess this one: https://www.elastic.co/guide/en/elasticsearch/reference/current/token-authentication-services.html